sdk/cards/src/internal/utils/PayloadBuilder.ts

```diff
@@ -16,53 +16,65 @@ import {
 } from "../Constants";
```

Comment:
add a return type to all methods wherever removed/missing

Reply:
@umangmoe reverted at all places where it was removed